{"name": "Login with valid phone number and invalid password -- @1.3 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//a[@data-csa-c-content-id='nav_ya_signin']\"}\n  (Session info: chrome=115.0.5790.111); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x0106A813+48355]\n\t(No symbol) [0x00FFC4B1]\n\t(No symbol) [0x00F05358]\n\t(No symbol) [0x00F309A5]\n\t(No symbol) [0x00F30B3B]\n\t(No symbol) [0x00F5E232]\n\t(No symbol) [0x00F4A784]\n\t(No symbol) [0x00F5C922]\n\t(No symbol) [0x00F4A536]\n\t(No symbol) [0x00F282DC]\n\t(No symbol) [0x00F293DD]\n\tGetHandleVerifier [0x012CAABD+2539405]\n\tGetHandleVerifier [0x0130A78F+2800735]\n\tGetHandleVerifier [0x0130456C+2775612]\n\tGetHandleVerifier [0x010F51E0+616112]\n\t(No symbol) [0x01005F8C]\n\t(No symbol) [0x01002328]\n\t(No symbol) [0x0100240B]\n\t(No symbol) [0x00FF4FF7]\n\tBaseThreadInitThunk [0x757100C9+25]\n\tRtlGetAppContainerNamedObjectPath [0x77077B1E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77077AEE+238]\n\n", "trace": "  File \"D:\\Documents\\pythonProject\\POC-\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"D:\\Documents\\pythonProject\\POC-\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login.py\", line 9, in step_impl\n    context.login_page = context.homepage.click_on_my_account()\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\home_page.py\", line 17, in click_on_my_account\n    self.click_on_element(\"my_account_option_xpath\", self.my_account_option_xpath)\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\BasePage.py\", line 28, in click_on_element\n    element = self.get_element(locator_type, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\BasePage.py\", line 42, in get_element\n    element = self.driver.find_element(By.XPATH, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Documents\\pythonProject\\POC-\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Documents\\pythonProject\\POC-\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\Documents\\pythonProject\\POC-\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I navigated to Login page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//a[@data-csa-c-content-id='nav_ya_signin']\"}\n  (Session info: chrome=115.0.5790.111); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x0106A813+48355]\n\t(No symbol) [0x00FFC4B1]\n\t(No symbol) [0x00F05358]\n\t(No symbol) [0x00F309A5]\n\t(No symbol) [0x00F30B3B]\n\t(No symbol) [0x00F5E232]\n\t(No symbol) [0x00F4A784]\n\t(No symbol) [0x00F5C922]\n\t(No symbol) [0x00F4A536]\n\t(No symbol) [0x00F282DC]\n\t(No symbol) [0x00F293DD]\n\tGetHandleVerifier [0x012CAABD+2539405]\n\tGetHandleVerifier [0x0130A78F+2800735]\n\tGetHandleVerifier [0x0130456C+2775612]\n\tGetHandleVerifier [0x010F51E0+616112]\n\t(No symbol) [0x01005F8C]\n\t(No symbol) [0x01002328]\n\t(No symbol) [0x0100240B]\n\t(No symbol) [0x00FF4FF7]\n\tBaseThreadInitThunk [0x757100C9+25]\n\tRtlGetAppContainerNamedObjectPath [0x77077B1E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77077AEE+238]\n\n", "trace": "  File \"D:\\Documents\\pythonProject\\POC-\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"D:\\Documents\\pythonProject\\POC-\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login.py\", line 9, in step_impl\n    context.login_page = context.homepage.click_on_my_account()\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\home_page.py\", line 17, in click_on_my_account\n    self.click_on_element(\"my_account_option_xpath\", self.my_account_option_xpath)\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\BasePage.py\", line 28, in click_on_element\n    element = self.get_element(locator_type, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\BasePage.py\", line 42, in get_element\n    element = self.driver.find_element(By.XPATH, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Documents\\pythonProject\\POC-\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Documents\\pythonProject\\POC-\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\Documents\\pythonProject\\POC-\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "attachments": [{"name": "screenshot", "source": "9440c789-9173-4e38-af51-94a67c851eb3-attachment.png", "type": "image/png"}], "start": 1691404853734, "stop": 1691404854009}, {"name": "When I enter valid phone number as \"8019713239\" and invalid password as \"Harish@AMAZON33\" into the fields", "status": "skipped", "start": 1691404857058, "stop": 1691404857058}, {"name": "And I click on Login button", "status": "skipped", "start": 1691404857058, "stop": 1691404857058}, {"name": "Then I should get a proper warning message on password page", "status": "skipped", "start": 1691404857058, "stop": 1691404857058}], "parameters": [{"name": "number", "value": "8019713239"}, {"name": "password", "value": "Harish@AMAZON33"}], "start": 1691404849768, "stop": 1691404857058, "uuid": "b894b84a-a920-4346-a824-69b399905021", "historyId": "9579b61a38b537dcda449bd87b6fa29d", "fullName": "Login Functionality: Login with valid phone number and invalid password", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "login"}, {"name": "feature", "value": "Login Functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}